### ISSUE TITLE
Iberdoex (Wordpress)

### ISSUE THREAD

**Issue by CloCkWeRX** [alltheplaces/alltheplaces#7498]
### Brand name

Iberdoex

Iberdoex is a group of service stations in the Autonomous Community of Extremadura.

### Wikidata ID

Q25689041
https://www.wikidata.org/wiki/Q25689041
https://www.wikidata.org/wiki/Special:EntityData/Q25689041.json

### Store finder url(s)

Official Url(s): https://iberdoex.com/
- [x] I have run the below and if anything in generated, added it.
- [ ] I have manually checked for a storefinder
- [ ] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy sf --brand-wikidata=Q25689041 https://iberdoex.com/`
```
from locations.storefinders.wp_store_locator import WPStoreLocatorSpider


class NewBrandZZSpider(WPStoreLocatorSpider):
        name = "new_brand_zz"
        allowed_domains = [
                "iberdoex.com",
        ]
```

**Comment by CloCkWeRX** [comment]
2024-02-29 10:49:25 [scrapy.spidermiddlewares.httperror] INFO: Ignoring response <400 https://iberdoex.com/wp-admin/admin-ajax.php?action=store_search&autoload=1>: HTTP status code is not handled or not allowed
2024-02-29 10:49:25 [scrapy.core.engine] INFO: Closing spider (finished)
2024-02-29 10:49:25 [locations.pipelines.duplicates] INFO: Dropped 0 duplicate items
2024-02-29 10:49:25 [scrapy.statscollectors] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 711,
 'downloader/request_count': 2,
 'downloader/request_method_count/GET': 2,
 'downloader/response_bytes': 903,
 'downloader/response_count': 2,
 'downloader/response_status_count/200': 1,
 'downloader/response_status_count/400': 1,
 'elapsed_time_seconds': 3.897728,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 2, 29, 10, 49, 25, 12620, tzinfo=datetime.timezone.utc),
 'httperror/response_ignored_count': 1,
 'httperror/response_ignored_status_count/400': 1,
 'log_count/DEBUG': 3,
 'log_count/INFO': 10,
 'memusage/max': 150581248,
 'memusage/startup': 150581248,
 'response_received_count': 2,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'start_time': datetime.datetime(2024, 2, 29, 10, 49, 21, 114892, tzinfo=datetime.timezone.utc)}
2024-02-29 10:49:25 [scrapy.core.engine] INFO: Spider closed (finished)
ubuntu@codespaces-db15fa:/workspaces/alltheplaces$

**Comment by CloCkWeRX** [comment]
Well, broken.
I think for now going to close as unscrapable.
